### ISSUE TITLE
Camera Type

### ISSUE THREAD

**Issue by rickyvetter** [socialtables/WebGLVM#2]
I've been playing around with Orthographic and Perspective cameras and they both seem to have advantages. Would we want to be able to switch between the two?

**Comment by tiansijie** [comment]
that's a good idea. I am also thinking switch the fixed top-down camera between the view(rotate) camera

**Comment by rickyvetter** [comment]
yeah. i agree. there could be two modes. one where its orthographic and just top-down (because rotating all of a sudden can be really surprising) and then a way to drop into a perspective camera that can rotate for viewing.

**Comment by tiansijie** [comment]
agreed
